packages/css/src/alert/alert.scss

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-alert {
+  align-items: flex-start;
+  background-color: var(--amsterdam-alert-background-color);
+  border-color: var(--amsterdam-alert-border-color);
+  border-style: var(--amsterdam-alert-border-style);
+  border-width: var(--amsterdam-alert-border-width);
```

Comment:
Is there a reason for not using the shorthand here?

Reply:
Can't we use the shorthand in the tokens as well? So:

`border: var(--amsterdam-alert-border);`

and

```
{
  "amsterdam": {
    "alert": {
      "border": { "value": "4px solid {amsterdam.color.yellow}" },
    }
  }
}
```

Reply:
The reason i did not do this is because error and success just set the border color and it might become confusing, especialy if you want to reuse our component.

Also, we realy need a semantic layer for color, something like `--amsterdam-color-reactive-error`.